cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -28,6 +28,7 @@
     protected static final List<String> CRUISE_CONTROL_GOALS_LIST = Collections.unmodifiableList(
         Arrays.asList(
                 CruiseControlGoals.RACK_AWARENESS_GOAL.toString(),
+                CruiseControlGoals.MIN_TOPIC_LEADERS_PER_BROKER_GOAL.toString(),
```

Comment:
Does the new goal need to be added also here: https://github.com/strimzi/strimzi-kafka-operator/blob/master/examples/cruise-control/kafka-rebalance-with-goals.yaml?